trl/trainer/ppo_trainer.py

```diff
@@ -301,6 +301,15 @@ def __init__(
         # init the current step
         self.current_step = 0
 
+        # init variables for pushing model to hub
+        if config.push_to_hub_if_best_kwargs:
+            assert (
+                "repo_id" in config.push_to_hub_if_best_kwargs
+            ), "You have to specify repo_id in order to push the model to the hub!"
```

Comment:
```suggestion
            if "repo_id" not in config.push_to_hub_if_best_kwargs:
                raise ValueError("You have to specify repo_id in order to push the model to the hub!")
```
Let's not use `assert` in the codebase! Why is that? I invite you to read https://snyk.io/blog/the-dangers-of-assert-in-python/ !

Reply:
Oh thanks!